### PR TITLE
[FLINK-18383][docs]translate "JDBC SQL Connector" documentation into …

### PR DESCRIPTION
**Pull request by JavaPentesters**
## What is the purpose of the change

+ This pull request is to translate "JDBC SQL Connector" documentation into [Chinese](https://issues.apache.org/jira/browse/FLINK-18383?page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel&focusedCommentId=17264916#comment-17264916)
+ The markdown file is located in flink/docs/dev/table/connectors/jdbc.zh.md

### CODE REVIEW

**Comment by wuchong** [comment]
Hi @leonardBang , could you help to review this?

**Comment by leonardBang** [comment]
I'd like to take a look this.

File: docs/dev/table/connectors/jdbc.zh.md [hunk 1]
```diff
@@ -92,210 +93,207 @@ ON myTopic.key = MyUserTable.id;
...
+
+特性
 --------
 
-### Key handling
+### 键处理
+
+Flink使用定义在DDL的主键写入数据到外部数据库。如果定义了主键，则连接器使用upsert 模式，否则连接器将使用append模式。
+
+在upsert模式下，Flink将根据主键插入一行或者更新已存在的记录，这种方式可以确保幂等性。为了确保输出预期的结果，我们推荐表要定义主键并且要确保主键是基础数据库表的唯一主键集或主键之一。在append模式下，Flink将把所有记录解释为插入消息，如果底层数据库发生了主键或者唯一约束的错误，INSERT插入可以失败。
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:262]:
```suggestion
在 upsert 模式下，Flink 根据主键插入新加的一行记录或者更新已存在的记录，这种方式可以确保幂等性。为了确保输出结果是符合预期的，推荐为表定义主键并且确保主键是底层数据库中表的主键集或唯一建。在 append 模式下，Flink 会把所有记录解释为插入消息，如果违反了底层数据库中主键或者唯一约束，INSERT 插入可能会失败。
```

File: docs/dev/table/connectors/jdbc.zh.md [hunk 2]
```diff
@@ -92,210 +93,207 @@ ON myTopic.key = MyUserTable.id;
...
 
-Flink uses the primary key that defined in DDL when writing data to external databases. The connector operate in upsert mode if the primary key was defined, otherwise, the connector operate in append mode.
+可以查看[CREATE TABLE DDL]({% link dev/table/sql/create.zh.md %}#create-table) 更多关于PRIMARY KEY语法。
 
-In upsert mode, Flink will insert a new row or update the existing row according to the primary key, Flink can ensure the idempotence in this way. To guarantee the output result is as expected, it's recommended to define primary key for the table and make sure the primary key is one of the unique key sets or primary key of the underlying database table. In append mode, Flink will interpret all records as INSERT messages, the INSERT operation may fail if a primary key or unique constraint violation happens in the underlying database.
+### 分区扫描
 
-See [CREATE TABLE DDL]({% link dev/table/sql/create.zh.md %}#create-table) for more details about PRIMARY KEY syntax.
+为了在 并行`Source` 任务实例下加速读取数据，Flink为JDBC table提供了分区扫描的特性。
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:268]: 为了在 并行的 `Source` task 实例中加速读取数据，Flink 为 JDBC 表提供了分区扫描的特性。

File: docs/dev/table/connectors/jdbc.zh.md [hunk 3]
```diff
@@ -92,210 +93,207 @@ ON myTopic.key = MyUserTable.id;
...
 
-### Partitioned Scan
+如果其中任一参数被指定的话，所有如下的扫描分区选项必须要指定，这些参数描述了在多个任务并行读取数据时如何对表进行分区。 `scan.partition.column` 必须是相关表中的数字、日期或时间戳列。注意 ，`scan.partition.lower-bound` 和 `scan.partition.upper-bound` 用于决定分区跨距和过滤表中的行。如果是批处理任务，也可以在提交flink任务之前先获取最大值和最小值：
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:270]:
```suggestion
下述所有的分区扫描选项必须要指定当其中任一参数被指定时，这些参数描述了在多个 task 并行读取数据时如何对表进行分区。 `scan.partition.column` 必须是相关表中的数字、日期或时间戳列。注意 ，`scan.partition.lower-bound` 和 `scan.partition.upper-bound` 用于决定分区的起始位置和过滤表中的数据。如果是批处理作业，也可以在提交 flink 作业之前获取最大值和最小值。
```

File: docs/dev/table/connectors/jdbc.zh.md [hunk 4]
```diff
@@ -92,210 +93,207 @@ ON myTopic.key = MyUserTable.id;
...
 
-To accelerate reading data in parallel `Source` task instances, Flink provides partitioned scan feature for JDBC table.
+- `scan.partition.column`: 输入用于进行分区的列名
+- `scan.partition.num`: 分区的数量
+- `scan.partition.lower-bound`: 第一个分区的最小值
+- `scan.partition.upper-bound`: 最后一个分区的最大值
 
-All the following scan partition options must all be specified if any of them is specified. They describe how to partition the table when reading in parallel from multiple tasks.
-The `scan.partition.column` must be a numeric, date, or timestamp column from the table in question. Notice that `scan.partition.lower-bound` and `scan.partition.upper-bound` are used to decide the partition stride and filter the rows in table. If it is a batch job, it also doable to get the max and min value first before submitting the flink job.
+### 查询缓存
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:277]: lookup cache

File: docs/dev/table/connectors/jdbc.zh.md [hunk 5]
```diff
@@ -92,210 +93,207 @@ ON myTopic.key = MyUserTable.id;
...
 
-- `scan.partition.column`: The column name used for partitioning the input.
-- `scan.partition.num`: The number of partitions.
-- `scan.partition.lower-bound`: The smallest value of the first partition.
-- `scan.partition.upper-bound`: The largest value of the last partition.
+JDBC连接器可以在临时表连接中用作查找源(例如维度表)，当前只支持同步的查找模式。默认情况下，查找缓存是未启用的，你可以通过设置`lookup.cache.max-rows` and `lookup.cache.ttl`等参数启用它。
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:279]:
```suggestion
JDBC 连接器可以用在时态表关联中作为一个可 lookup 的 Source (又称为维表)，当前只支持同步的查找模式。
默认情况下，lookup cache 是未启用的，你可以设置 `lookup.cache.max-rows` and `lookup.cache.ttl` 参数来启用。
```

File: docs/dev/table/connectors/jdbc.zh.md [hunk 6]
```diff
@@ -92,210 +93,207 @@ ON myTopic.key = MyUserTable.id;
...
 
-### Lookup Cache
+查找缓存的主要目的是用于提高JDBC连接器临时表连接的性能。默认情况下，因为没有启用查找缓存，所以所有请求都被发送到外部的数据库。当查找缓存被启用时，每个进程(如TaskManager)将保存一个缓存。Flink将优先查找缓存，只有当缓存未查找到时才向外部数据库发送请求，并使用返回的行更新缓存。
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:281]: lookup cache 的主要目的是用于提高时态表关联 JDBC 连接器的性能。默认情况，lookup cache 不开启，所以所有请求都会发送到外部的数据库。当 lookup cache 被启用时，每个进程（即 TaskManager ）将维护一个缓存。Flink 将优先查找缓存，只有当缓存未查找到时才向外部数据库发送请求，并使用返回的数据更新缓存。

File: docs/dev/table/connectors/jdbc.zh.md [hunk 7]
```diff
@@ -92,210 +93,207 @@ ON myTopic.key = MyUserTable.id;
...
-The cached rows might not be the latest, users can tune `lookup.cache.ttl` to a smaller value to have a better fresh data, but this may increase the number of requests send to database. So this is a balance between throughput and correctness.
 
-### Idempotent Writes
+### 幂等写入
 
-JDBC sink will use upsert semantics rather than plain INSERT statements if primary key is defined in DDL. Upsert semantics refer to atomically adding a new row or updating the existing row if there is a unique constraint violation in the underlying database, which provides idempotence.
+如果在DDL中定义了主键，JDBC sink将使用upsert语义而不是普通的INSERT语句。Upsert语义指的是如果底层数据库中存在违反唯一性约束，则原子地添加新行或更新现有行，这种方式了提供了幂等性。
 
-If there are failures, the Flink job will recover and re-process from last successful checkpoint, which can lead to re-processing messages during recovery. The upsert mode is highly recommended as it helps avoid constraint violations or duplicate data if records need to be re-processed.
+如果出现故障，Flink任务将从上次成功的检查点(checkpoint)恢复并重新处理，这可能导致在恢复期间重新处理消息。我们强烈推荐使用upsert模式，因为如果需要重新处理记录，它有助于避免违反约束或产生重复数据。
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:292]: 如果出现故障，Flink 作业会从上次成功的 checkpoint 恢复并重新处理，这可能导致在恢复过程中重复处理消息。强烈推荐使用 upsert 模式，因为如果需要重复处理记录，它有助于避免违反数据库主键约束和产生重复数据。

File: docs/dev/table/connectors/jdbc.zh.md [hunk 8]
```diff
@@ -92,210 +93,207 @@ ON myTopic.key = MyUserTable.id;
...
 
-Aside from failure recovery, the source topic may also naturally contain multiple records over time with the same primary key, making upserts desirable.
+除了故障恢复之外，源主题（the source topic）还可能随着时间的推移自然地包含多个具有相同主键的记录，因此需要使用upsert。
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:295]: 除了故障恢复场景外，数据源（kafka topic）也可能随着时间的推移自然地包含多个具有相同主键的记录，这使得 upsert 模式是用户期待的。

File: docs/dev/table/connectors/jdbc.zh.md [hunk 9]
```diff
@@ -92,210 +93,207 @@ ON myTopic.key = MyUserTable.id;
...
 
-As there is no standard syntax for upsert, the following table describes the database-specific DML that is used.
+由于upsert没有标准语法，因此下表描述了所使用的特定于数据库的DML：
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:296]:
```suggestion
由于 upsert 没有标准的语法，因此下表描述了不同数据库的 DML 语法：
```

File: docs/dev/table/connectors/jdbc.zh.md [hunk 10]
```diff
@@ -316,11 +314,9 @@ As there is no standard syntax for upsert, the following table describes the dat
     </tbody>
 </table>
 
-### Postgres Database as a Catalog
+### Postgres数据库作为目录
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:317]:
```suggestion
### Postgres 数据库作为 Catalog
```
Catalog 作为术语翻译

File: docs/dev/table/connectors/jdbc.zh.md [hunk 11]
```diff
@@ -332,18 +328,18 @@ PostgresCatalog.getTable(ObjectPath tablePath)
 PostgresCatalog.tableExists(ObjectPath tablePath)
 {% endhighlight %}
 
-Other `Catalog` methods is unsupported now.
+其他的 `Catalog` 方式现在还是不支持的。
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:331]: 方式 -> 方法

File: docs/dev/table/connectors/jdbc.zh.md [hunk 12]
```diff
@@ -332,18 +328,18 @@ PostgresCatalog.getTable(ObjectPath tablePath)
...
 
-#### Usage of PostgresCatalog
+#### 使用PostgresCatalog
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:333]: #### PostgresCatalog 的使用

File: docs/dev/table/connectors/jdbc.zh.md [hunk 13]
```diff
@@ -332,18 +328,18 @@ PostgresCatalog.getTable(ObjectPath tablePath)
...
 
-Please refer to [Dependencies](#dependencies) section for how to setup a JDBC connector and Postgres driver.
+你可以查阅[Dependencies](#dependencies) 部分了解如何建立JDBC连接器和Postgres驱动。
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:335]:
```suggestion
请参考[依赖](#依赖) 章节了解如何配置 JDBC 连接器和 Postgres 驱动。
```

File: docs/dev/table/connectors/jdbc.zh.md [hunk 14]
```diff
@@ -332,18 +328,18 @@ PostgresCatalog.getTable(ObjectPath tablePath)
...
-- `default-database`: required, default database to connect to.
-- `username`: required, username of Postgres account.
-- `password`: required, password of the account.
-- `base-url`: required, should be of format `"jdbc:postgresql://<ip>:<port>"`, and should not contain database name here.
+Postgres catalog支持以下参数:
+- `name`: 必填, catalog的名称
+- `default-database`:  必填, 默认要连接的数据库
+- `username`: 必填, Postgres 账户的用户名
+- `password`: 必填, 账户的密码
+- `base-url`: 必填, 应该符合格式 `"jdbc:postgresql://<ip>:<port>"`, 并且这里应该不包含数据库名称
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:342]:
```suggestion
- `base-url`: 必填, 应该符合`"jdbc:postgresql://<ip>:<port>"`的格式, 同时这里不应该包含数据库名。
```

File: docs/dev/table/connectors/jdbc.zh.md [hunk 15]
```diff
@@ -393,10 +390,11 @@ val baseUrl         = "..."
 val catalog = new JdbcCatalog(name, defaultDatabase, username, password, baseUrl)
 tableEnv.registerCatalog("mypg", catalog)
 
-// set the JdbcCatalog as the current catalog of the session
+// 设置JdbcCatalog为会话的当前目录
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:393]:
```suggestion
// 设置 JdbcCatalog 为会话的当前 catalog
```

File: docs/dev/table/connectors/jdbc.zh.md [hunk 16]
```diff
@@ -393,10 +390,11 @@ val baseUrl         = "..."
...
 tableEnv.useCatalog("mypg")
 {% endhighlight %}
 </div>
+
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:397]: useless blank line, the html tag `div` can format this section well.

File: docs/dev/table/connectors/jdbc.zh.md [hunk 17]
```diff
@@ -437,22 +436,23 @@ catalogs:
 </div>
 </div>
 
-#### PostgresSQL Metaspace Mapping
+#### PostgresSQL 元空间映射
+
+除了database之外，postgresSQL还有一个额外的命名空间`schema`。一个Postgres实例可以拥有多个数据库；每个数据库可以拥有多个模式，其中一个模式默认名为“public”；每个模式可以包含多个表。
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:441]: 除了数据库之外，postgresSQL 还有一个额外的命名空间 `模式`。一个 Postgres 实例可以拥有多个数据库， 每个数据库可以拥有多个模式，其中一个模式默认名为“public”，每个模式可以包含多张表。

File: docs/dev/table/connectors/jdbc.zh.md [hunk 18]
```diff
@@ -467,9 +467,9 @@ SELECT * FROM mydb.`custom_schema.test_table2`;
 SELECT * FROM `custom_schema.test_table2`;
 {% endhighlight %}
 
-Data Type Mapping
+数据类型映射
 ----------------
-Flink supports connect to several databases which uses dialect like MySQL, PostgresSQL, Derby. The Derby dialect usually used for testing purpose. The field data type mappings from relational databases data types to Flink SQL data types are listed in the following table, the mapping table can help define JDBC table in Flink easily.
+Flink支持连接到多个使用方言(dialect)的数据库，如MySQL、PostgresSQL、Derby等。其中，Derby方言通常是用于测试目的。下表列出了从关系数据库数据类型到Flink SQL数据类型的字段数据类型映射，映射表可以帮助您轻松地在Flink中定义JDBC表。
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:472]:
```suggestion
Flink支持连接到多个使用方言（dialect）的数据库，如 MySQL、PostgresSQL、Derby 等。其中，Derby 通常是用于测试目的。下表列出了从关系数据库数据类型到 Flink SQL 数据类型的类型映射，映射表可以使得在 Flink 中定义 JDBC 表更加简单。
```

**Comment by JavaPentesters** [comment]
@leonardBang Thanks for all your great suggestion! It's my first time to receive PR review.So, what should I do next? How to append commit?

**Comment by leonardBang** [comment]
> @leonardBang Thanks for all your great suggestion! It's my first time to receive PR review.So, what should I do next? How to append commit?

You can append a commit to address the comments, you can directly push code to your remote branch(git push origin JavaPentesters) after appended commit in your local branch (`JavaPentesters:master`) , the PR will update automatically.

File: docs/dev/table/connectors/jdbc.zh.md [hunk 19]
```diff
@@ -92,210 +93,207 @@ ON myTopic.key = MyUserTable.id;
...
+      <td>最大重试超时时间，超时应该是以秒为粒度的，并且不应该小于1秒</td>
     </tr>
     <tr>
       <td><h5>scan.partition.column</h5></td>
-      <td>optional</td>
+      <td>可选</td>
       <td style="word-wrap: break-word;">(none)</td>
       <td>String</td>
-      <td>The column name used for partitioning the input. See the following <a href="#partitioned-scan">Partitioned Scan</a> section for more details.</td>
+      <td>输入用于进行分区的列名，有关更多细节请参阅下面的 <a href="#partitioned-scan">分区扫描</a>部分</td>
```
[the remark this answers is not in the document]
Comment (JavaPentesters) [reply docs/dev/table/connectors/jdbc.zh.md:164]: <td>用于将输入进行分区的列名，请参阅下面的 <a href="#partitioned-scan">分区扫描</a>部分了解更多详情。</td>

**Comment by JavaPentesters** [comment]
> > @leonardBang Thanks for all your great suggestion! It's my first time to receive PR review.So, what should I do next? How to append commit?
> 
> You can append a commit to address the comments, you can directly push code to your remote branch(git push origin JavaPentesters) after appended commit in your local branch (`JavaPentesters:master`) , the PR will update automatically.

Thanks for your directions! I have appended a commit to fix this issues.Looking forward to seeing you to help me to review this again!

File: docs/dev/table/connectors/jdbc.zh.md [hunk 20]
```diff
@@ -31,35 +31,35 @@ under the License.
...
 
-Dependencies
+
+依赖
 ------------
 
 {% assign connector = site.data.sql-connectors['jdbc'] %} 
 {% include sql-connector-download-table.zh.html 
-    connector=connector
+​    connector=connector
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:44]: revert the change, otherwise the doc build will fail

File: docs/dev/table/connectors/jdbc.zh.md [hunk 21]
```diff
@@ -31,35 +31,35 @@ under the License.
...
-| PostgreSQL  |  `org.postgresql`  |      `postgresql`      | [Download](https://jdbc.postgresql.org/download.html) |
-| Derby       | `org.apache.derby` |        `derby`         | [Download](http://db.apache.org/derby/derby_downloads.html) |
+| MySQL       |       `mysql`      | `mysql-connector-java` | [下载](https://repo.maven.apache.org/maven2/mysql/mysql-connector-java/) |
+| PostgreSQL  |  `org.postgresql`  |      `postgresql`      | [下载](https://jdbc.postgresql.org/download.html) |
+| Derby       | `org.apache.derby` |        `derby`         | [下载](http://db.apache.org/derby/derby_downloads.html) |
 
 <br>
-JDBC connector and drivers are not currently part of Flink's binary distribution. See how to link with them for cluster execution [here]({% link dev/project-configuration.zh.md %}).
-
+当前，JDBC 连接器和驱动在 Flink 二进制发布包中，请参阅 [这里]({% link dev/project-configuration.zh.md %}) 了解在集群上执行时何连接它们。
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:57]:
```suggestion
当前，JDBC 连接器和驱动不在 Flink 二进制发布包中，请参阅 [这里]({% link dev/project-configuration.zh.md %}) 了解在集群上执行时何连接它们。
```

File: docs/dev/table/connectors/jdbc.zh.md [hunk 22]
```diff
@@ -31,35 +31,35 @@ under the License.
...
 
-How to create a JDBC table
+如何创建JDBC table
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:59]: （1）英文单词前后是中文时，前后都需要有空格，建议全文检查下

File: docs/dev/table/connectors/jdbc.zh.md [hunk 23]
```diff
@@ -92,210 +92,206 @@ ON myTopic.key = MyUserTable.id;
...
-      <td>Defines the parallelism of the JDBC sink operator. By default, the parallelism is determined by the framework using the same parallelism of the upstream chained operator.</td>
+      <td>用于定义 JDBC sink 算子的并行度。默认情况下，并行度是由框架决定的：使用与上游链式算子相同的并行度。</td>
     </tr>
     </tbody>
 </table>
 
-Features
+
+特性
 --------
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:255]: 本地编译下文档，先检查下，目前格式是乱掉的
步骤为：（1）cd flink/docs   （2）./build_docs.sh -p （3）在本地4000 端口查看
如果装了docker，也可以(1)cd flink/docs/docker (2) ./run.sh (3) 用docker build，(4) 在本地4000端口查看
image

File: docs/dev/table/connectors/jdbc.zh.md [hunk 24]
```diff
@@ -92,210 +92,206 @@ ON myTopic.key = MyUserTable.id;
...
-      <td>The number of partitions.</td>
+      <td>分区的数量</td>
     </tr>
     <tr>
       <td><h5>scan.partition.lower-bound</h5></td>
-      <td>optional</td>
+      <td>可选</td>
       <td style="word-wrap: break-word;">(none)</td>
       <td>Integer</td>
-      <td>The smallest value of the first partition.</td>
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:177]: 英文都有句号，翻译时全部都保留吧，后面的表格中长句都有句号，统一点。

**Review by leonardBang — COMMENTED** [review]
@JavaPentesters Thanks for the update, I left some comments

File: docs/dev/table/connectors/jdbc.zh.md (hunk 20 above)
Comment (JavaPentesters) [reply docs/dev/table/connectors/jdbc.zh.md:44]: Thanks! Got it!

**Comment by JavaPentesters** [comment]
@leonardBang Thanks for your review! All the mentioned comments have been solved! And, The documentation can be translated to HTML!

File: docs/dev/table/connectors/jdbc.zh.md [hunk 25]
```diff
@@ -31,35 +31,33 @@ under the License.
...
+| PostgreSQL  |  `org.postgresql`  |      `postgresql`      | [下载](https://jdbc.postgresql.org/download.html) |
+| Derby       | `org.apache.derby` |        `derby`         | [下载](http://db.apache.org/derby/derby_downloads.html) |
 
 <br>
-JDBC connector and drivers are not currently part of Flink's binary distribution. See how to link with them for cluster execution [here]({% link dev/project-configuration.zh.md %}).
-
+当前，JDBC 连接器和驱动不在 Flink 二进制发布包中，请参阅 [这里]({% link dev/project-configuration.zh.md %}) 了解在集群上执行时何连接它们。
 
-How to create a JDBC table
+如何创建 JDBC table
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:57]:
```suggestion
如何创建 JDBC 表
```

File: docs/dev/table/connectors/jdbc.zh.md [hunk 26]
```diff
@@ -92,210 +90,206 @@ ON myTopic.key = MyUserTable.id;
 </div>
 </div>
 
-Connector Options
+连接器参数选项
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:93]:
```suggestion
连接器参数
```

File: docs/dev/table/connectors/jdbc.zh.md [hunk 27]
```diff
@@ -92,210 +90,206 @@ ON myTopic.key = MyUserTable.id;
...
 
 <table class="table table-bordered">
     <thead>
       <tr>
-        <th class="text-left" style="width: 25%">Option</th>
-        <th class="text-left" style="width: 8%">Required</th>
-        <th class="text-left" style="width: 7%">Default</th>
-        <th class="text-left" style="width: 10%">Type</th>
-        <th class="text-left" style="width: 50%">Description</th>
+        <th class="text-left" style="width: 25%">选项</th>
```
Comment (leonardBang) [inline docs/dev/table/connectors/jdbc.zh.md:99]:
```suggestion
        <th class="text-left" style="width: 25%">参数</th>
```

Comment (leonardBang) [reply docs/dev/table/connectors/jdbc.zh.md:99]: 和其他中文文档保持一致吧

**Review by leonardBang — APPROVED** [review]
Thanks for the quick update, @JavaPentesters The PR generally look good to me, I left minor comments, could you squash you changes to one commit and rebase to the branch to latest master?

**Comment by leonardBang** [comment]
@JavaPentesters  Looks like you're using `git merge command`, it's recommended to use `git rebase master`.   You can use IDE version control util to manage your commit easily
